moveit_setup_assistant/include/moveit/setup_assistant/tools/moveit_config_data.h

```diff
@@ -277,7 +269,8 @@ class MoveItConfigData
   bool outputKinematicsYAML(const std::string& file_path);
   bool outputJointLimitsYAML(const std::string& file_path);
   bool outputFakeControllersYAML(const std::string& file_path);
-  void outputFollowJointTrajectoryYAML(YAML::Emitter& emitter);
+  void outputFollowJointTrajectoryYAML(YAML::Emitter& emitter,
+                                       std::vector<ROSControlConfig>* ros_controllers_config_copy);
```

Comment:
why is this called ``_copy``?

Reply:
Because this function modifies the `ros_controllers_config_` object, we don't want to modify the original object here.